src/KSPACE/fft3d.cpp

```diff
@@ -103,18 +103,18 @@ void fft_3d(FFT_DATA *in, FFT_DATA *out, int flag, struct fft_plan_3d *plan)
   length = plan->length1;
 
 #if defined(FFT_MKL)
-  if (flag == -1)
+  if (flag == 1)
```

Comment:
@sjplimp This change and the following below break all PPPM styles. Am reverting it.

Reply:
ok - will take a look - thanks